### ISSUE TITLE
[Story] Move upload buttons from "Create" button context menu to "Upload" button context menu

### ISSUE THREAD

**Issue by shepilov** [linagora/twake-drive-legacy#226]
## User story summary
Right now we can upload files in multiple ways:
- drag'n'drop
- upload button(only files)
- create menu -> upload button (files and folder)

## Goal
Remove "upload ... " items from "Create" menu context menu
And context menu on the "Upload" button with two items:
- Folder
- Files

**Comment by tibormichel1** [comment]
Hello can i work on this issue ?

**Comment by shepilov** [comment]
@tibormichel1 sure

**Comment by tibormichel1** [comment]
@shepilov Okay, thanks! could you also asign it to me ? I need to get assigned and finish issue as part of my university task

**Comment by tibormichel1** [comment]
Hello @shepilov i tried quite a lot and did a research and i found out it is not possible to use one button to upload files and directories i checked also on few cloud services eg. google drive and they also do not support this functionality you have allways to choose if you want to upload dirs or files do you have any hint how to solve it ?

**Comment by shepilov** [comment]
@tibormichel1 yes, it looks like you are right, there is no way to have universal input and the same behavior as for drag'n'drop.
I'll change the user story to move the two menu items: "upload files", and "upload folder" to the new context menu that appears when you click on "upload" button

**Comment by tibormichel1** [comment]
@shepilov  Okay thanks, i was thinking of suggesting you the same option you offered now :)
